File: app/src/main/java/com/wolkowiczmateusz/android/cleanmvpretrofitrxdaggerboilerplate/data/api/CustomApi.java
package com.wolkowiczmateusz.android.cleanmvpretrofitrxdaggerboilerplate.data.api;

import com.wolkowiczmateusz.android.cleanmvpretrofitrxdaggerboilerplate.data.api.response.UserResponse;

import io.reactivex.Observable;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface CustomApi {

    @GET("login")
    Observable<UserResponse> getLogin(@Query("username") String username, @Query("password") String password);
}

File: app/src/main/java/com/wolkowiczmateusz/android/cleanmvpretrofitrxdaggerboilerplate/data/api/CustomApi.kt
package com.wolkowiczmateusz.android.cleanmvpretrofitrxdaggerboilerplate.data.api

import com.wolkowiczmateusz.android.cleanmvpretrofitrxdaggerboilerplate.data.api.response.UserResponse

import io.reactivex.Observable
import retrofit2.http.GET
import retrofit2.http.Query

interface CustomApi {

    @GET("login")
    fun getLogin(@Query("username") username: String, @Query("password") password: String): Observable<UserResponse>
}
